### ISSUE TITLE
Busts not displaying

### ISSUE THREAD

**Issue by Fenoxo** [OXOIndustries/Trials-in-Tainted-Space#800]
I'm noticing that busts aren't displaying when I encounter cock vine seedlings. It calls the header function (which works elsewhere), and there are no more clearOutput()s after. Not sure what's causing it. Did anyone fuck with bust code lately?

**Comment by Fenoxo** [comment]
Here's a line I'm seeing when I meet them: if(pc.reflexes()/2 + rand(20) + 1 < 13) output(" You almost jump out of your skin when something touches your [pc.legOrLegs], and you curse as you hear some small pest slithering swiftly away from you in the darkness. Caves are just the goddamn worst.");

**Comment by Fenoxo** [comment]
Lys also appears not to show her bust sometimes:http://puu.sh/p5uct/4f2c7fce64.png

**Comment by Jacques00** [comment]
That's weird. The only thing I touched related to busts is for when they are cleared and when they are persistent--like the busts used to clear out when clicking on a "disabled" button, but I added a check for that and when going into any of the side menus (main menu, data, level up, etc.) and auto clearing when new buffer messages show up. Otherwise, the busts should perform like normal.

I tried testing Lys and the seedling and they all seem to show up most, if not 100%, of the time. Is there something specific you are doing before running into them or does this issue occur at any moment for you?
